notebooks/304-pytorch-post-training-quantization/Readme.md

```diff
@@ -0,0 +1,18 @@
+# Optimizing PyTorch models with Neural Network Compression Framework of OpenVINO by 8-bit quantization.
+
+This tutorial demonstrates how to use [NNCF](https://github.com/openvinotoolkit/nncf) 8-bit quantization to optimize the 

```

Comment:
Let's improve it and clearly state that the notebook demonstrates how to use NNCF at post-training, i.e. without the training pipeline.

Reply:
Done